media-sound/freac/patches/freac-1.1~alpha20180913.patchset

```diff
@@ -0,0 +1,106 @@
+From 3a245e17b789774bd3419322855e79a2bb7ff044 Mon Sep 17 00:00:00 2001
+From: Robert Kausch <robert.kausch@freac.org>
+Date: Fri, 5 Oct 2018 21:44:28 +0000
+Subject: Add fre:ac resource definition file.
```

Comment:
Although we still have many recipes where the rdef file is in the patchset, we tend to prefer having them moved to the `additional-files/` subdirectory.
(You can have a look at how we do this with e.g.  media-video/vlc/. It requires defining a ADDITIONAL_FILES variable.)

Reply:
@fbrosson Thanks for your comments and suggestions! I addressed them all in the latest commit. smooth and BoCA now have separate PRs #3198 and #3199.